specs/method-identifiers.html

```diff
@@ -142,294 +115,34 @@
-          <p>There is an open issue about whether payment method identifiers should resolve to a resource if they are URLs.</p>
-        </div>
-      </section>
-    </section>
-
-    <section>
-      <h2>Option 1b</h2>
-      <p>
-        This section describes a way of expressing payment method identifiers
-        using machine-readable URLs that may be aliased to
-        developer-friendly short identifiers.
-      </p>
-
-      <section>
-        <h2>Identifier format</h2>
-        <p>
-          Payment method identifiers are
-          <a data-lt="absolute URL">absolute URLs</a> that may optionally be
-          aliased using short identifiers. For example,
-          <code>https://visa.com/payment-methods/VisaDebit</code> (URL) or
-          <code>VisaDebit</code> (short identifier).
-        </p>
-      </section>
-
-      <section>
-        <h2>Short Identifier Registry</h2>
-        <p>
-          A mapping file establishing short identifiers to payment method
-          identifier URLs can be found at
-          <code>https://w3.org/registries/web-payments/v1</code>.
-          To reduce the load on fetching this file, it is encouraged that
-          applications hard code the mappings.
-        </p>
-        <p class="issue">
-          Any short identifier registry that is not re-parsed every time
-          will require a message version identifier to be embedded in the
-          messages that use the short identifiers. The assumption here is
-          that there is some versioning mechanism that is used by payment
-          messages such that the payment apps and merchants know how to
-          process short identifiers.
-        </p>
-        <p class="issue">
-          The format of the document located at the short identifier registry
-          URL above still needs to be determined. Options include a plain
-          JSON document or a JSON-LD document. The mappings would be as simple
-          as: <code>"VisaDebit": "https://visa.com/payment-methods/VisaDebit"</code>.
-        </p>
-        <p class="issue">
-          It is assumed that once the short identifier registry file is created,
-          it is never updated. The Working Group may periodically release
-          <code>v2</code>, <code>v3</code>, etc. registry files.
-        </p>
-      </section>
-
-      <section>
-        <h2>Content at Payment Method Identifier URLs</h2>
-        <p>
-          Payment method identifier URLs that resolve to content:
-        </p>
-        <ol>
-          <li>MUST be served over HTTPS,</li>
-          <li>SHOULD support HTTP Content Negotiation,</li>
-          <li>SHOULD provide a human-readable document that describes the
-          payment method when <code>text/html</code> is requested, and</li>
-          <li>SHOULD provide a machine-readable document that describes the
-          payment method when <code>application/ld+json</code> is requested.
-          </li>
-        </ol>
-      </section>
```

Comment:
This section on information for payment method identifiers that do happen to resolve to content shouldn't be completely removed just because we settled on using absolute URLs. This is still an additional option under consideration.